### DISCUSSION TITLE
Components remain WAIT_START state

### DISCUSSION THREAD

**Discussion by bnurrr** [Security-Onion-Solutions/securityonion#8058] [Unsupported Versions] [answered]
Hi, We have Security onion 2.3.120 distributed deployment; a manager node, a search node, a sensor node and a fleet node.
I have a problem, I restart hosts all components remain WAIT_START state, except Docker, I have to do restart all  manually, what can i do this case and how start all services automaticly and how restart hosts securely

**Comment by dougburks** [comment]
Do you see any errors if you run `sudo salt-call state.highstate`?

Have you modified your configuration? If so, have you double-checked for typos or formatting errors?

**Comment by bnurrr** [comment]
Hi,

```
sudo salt-call state.highstate

local:
    Data failed to compile:
----------
    The function "state.sls" is running as PID 57613 and was started at 2022, Jun 04 05:57:50.315469 with jid 20220604055750315469
```

No, I didn't any change but a week ago virtual server's host shutdown unexpectedly.
I start components manually on manager node  but I can't start so-sensoroni. Fleet, search and sensor node only docker state OK other's WAIT_START,  and I can't start these.

```
sudo so-filebeat-start

Attempt to authenticate with the salt master failed with timeout error
```
```

 sudo salt "*" cmd.run "so-status"

_sensor:
    Minion did not return. [No response]
    The minions may not have all finished running and any remaining minions will return upon completion. To look up the return                                                                 data for this job later, run the following command:

    salt-run jobs.lookup_jid 20220604062111430490
_fleet:
    Minion did not return. [No response]
    The minions may not have all finished running and any remaining minions will return upon completion. To look up the return                                                                 data for this job later, run the following command:

    salt-run jobs.lookup_jid 20220604062111430490
_searchnode:
    Minion did not return. [No response]
    The minions may not have all finished running and any remaining minions will return upon completion. To look up the return                                                                 data for this job later, run the following command:

    salt-run jobs.lookup_jid 20220604062111430490
_idh:
    Minion did not return. [No response]
    The minions may not have all finished running and any remaining minions will return upon completion. To look up the return                                                                 data for this job later, run the following command:

    salt-run jobs.lookup_jid 20220604062111430490
ERROR: Minions returned with non-zero exit code
```

**Comment by dougburks** [comment]
Can the nodes connect to the salt ports on the manager?
https://docs.securityonion.net/en/2.3/firewall.html#node-communication

**Reply by bnurrr** [reply]
Yes, all host at same vlan and not any restriction  between them

**Reply by dougburks** [reply]
What is the output of the following?
```
sudo salt-key -L
```

Have you checked the salt logs in `/opt/so/log/salt/` for additional clues?

**Comment by bnurrr** [comment, marked as answer]
Hi,

Sorry for being late,

`Accepted Keys:
..._fleet
..._idh
..._manager
..._searchnode
..._sensor
Denied Keys:
Unaccepted Keys:
Rejected Keys:`

and in  /opt/so/log/salt/ directory 

in master-20220616.gz file shows below error

` [salt.utils.event :908 ][ERROR   ][1837] Event iteration failed with exception: 'list' object has no attribute 'items'`

I guess I will setup 2.3.130 version
